### PR TITLE
cleanup of gap3 interface file

### PR DESCRIPTION
**Pull request by fchapoton**
This refreshes the modified file, mostly for pep8 compliance

also gathering doctests annotations

also changes two calls to `super`

### :memo: Checklist

- [x] The title is concise, informative, and self-explanatory.
- [x] The description explains in detail what this PR is about.

### CODE REVIEW

File: src/sage/interfaces/gap3.py [hunk 1]
```diff
@@ -510,20 +519,21 @@ def cputime(self, t=None):
 
         EXAMPLES::
 
-            sage: t = gap3.cputime()                       #optional - gap3
-            sage: t  #random                               #optional - gap3
+            sage: # optional - gap3
+            sage: t = gap3.cputime()
+            sage: t  #random
             0.02
```
Comment (kwankyu) [inline src/sage/interfaces/gap3.py:529]: `# random`

Comment (kwankyu) [reply src/sage/interfaces/gap3.py:529]: Two more below.

**Comment by kwankyu** [comment]
Otherwise, lgtm.

**Comment by kwankyu** [comment]
Was I too fast? Does it need review?

**Comment by fchapoton** [comment]
I still need to check that it works with gap3 installed. The CI (if it works, which is not clear) should check without gap3.

**Comment by kwankyu** [comment]
OK. I checked without gap3.

**Review by kwankyu — APPROVED** [review]

**Pull request merged**